css/css-fonts/font-feature-settings-descriptor-01.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+<title>CSS Test: font-feature-settings descriptor</title>
+<link rel="author" title="Chris Lilley" href="chris@w3.org">
+<link rel="help" href="https://www.w3.org/TR/css-fonts-3/#font-rend-desc">
+<link rel="help" href="https://www.w3.org/TR/css-fonts-3/#font-feature-settings-prop">
+<link rel="match" href="font-feature-settings-descriptor-01-ref.html">
+<meta name="assert" content="These descriptors define initial settings that apply when the font defined by an @font-face rule is rendered">
+<style>
+  @font-face {
+    font-family: fwf;
+    src: url(support/fonts/FontWithFancyFeatures.otf);
+  }
+  @font-face {
+    font-family: fwf2;
+    src: url(support/fonts/FontWithFancyFeatures.otf);
+    font-feature-settings: "liga" on, "clig" on, "calt" on, "hlig" on, "dlig" on, "onum" on, "smcp" on, "jp90" on;
+  }
+  .test {
+	  font-family: fwf;
+	  font-size: 2.4em;
+	  line-height: 1.1;
+  }
+  .test>p.descriptor {
+      font-family: fwf2;
+  }
+  .property {
+    font-feature-settings: "liga" on, "clig" on, "calt" on, "hlig" on, "dlig" on, "onum" on, "smcp" on, "jp90" on;
+  }
+</style>
+
+<body>
+<p>Test passes if the three lines below are identical, with eight check marks (✓). </p>
+<section class="test">
+    <p class="descriptor">CDGFEJQa</p>
```

Comment:
I think it would be useful to add a comment of how this test works mechanically in combination with the FontWithFancyFeatures font. I assume it does because the ligatures features substitue CFGFEJQa into A's which then comes out as check marks again?

Reply:
PS: Thanks for contributing this useful test case.

Reply:
It works the same as all the other tests which use that font (there are a lot of them).

For each character in the font (apart from A and B) the resulting glyph is a checkmark if a particular opentype feature is on, and a cross if it is off. A and B are hardcoded to checkmark and cross, respectively, so are useful for reftests.

Documentation [here](https://www.w3.org/People/chris/fwf/index.html) and specific details of [which opentype feature goes with which test](https://www.w3.org/People/chris/fwf/fancytester.html)

Reply:
The font was made by @litherum by the way, which is why I tagged him as a reviewer